R/checkSummations.R

```diff
@@ -258,9 +248,20 @@ checkSummations <- function(mifFile, outputDirectory = ".", template = NULL, sum
         }
         if (generatePlots) {
           message("Add plot for ", p)
-          mip::showAreaAndBarPlots(plotdata, intersect(childs, unique(plotdata$variable)),
-                                   tot = gsub(" [1-9]$", "", p),
-                                   mainReg = mainReg, yearsBarPlot = c(2030, 2050), scales = "fixed")
+
+          s <- summationGroups %>%
+            filter(.data$parent == p) %>%
+            select(c("child", "factor"))
+
+          df <- data %>%
```

Comment:
Thank you, really helpful.

I think it might be worth adjusting the unit as well. If you look at Renato's plot, the variables all have different unit, when in fact, the they are converted here into the unit of the parent (or the check doesn't make sense at all). So I would suggest to add a `mutate("unit" := filter(plotdata, .data$variable == p)$unit[[1]])` (untested) or something like that to have a single unit there.
![image](https://github.com/pik-piam/piamInterfaces/assets/90761609/dbb8c304-4225-4e59-b77f-1817ca2a6302)

Reply:
Done